src/server.ts

```diff
@@ -0,0 +1,7 @@
+import { FastifyInstance } from 'fastify';
+
+export function runServer(app: FastifyInstance, port: number) {
+  app.listen({ port }).then(() => {
+    console.log(`The server is running on port ${port}`);
+  });
+}
```

Comment:
```suggestion
export async function runServer(app: FastifyInstance, port: number) {
  await app.listen({ port });
  console.log(`The server is running on port ${port}`);
}
```

Com isso, nós garantimos que ao usar essa função, desde que utilizada com async/await, a gente espera o server rodar para prosseguir, e não arriscamos alguma tarefa começar antes dela terminar.